src/features/user/ManageProjects/components/ProjectMedia.tsx

```diff
@@ -292,7 +298,7 @@ export default function ProjectMedia({
                       onBlur={(e) => uploadCaption(image.id, index, e)}
                       type="text"
                       placeholder={t('manageProjects:addCaption')}
-                      defaultValue={image.description}
+                      defaultValue={image.description ? '' : undefined}
```

Comment:
In what scenario would you want this to be `undefined`? Wouldn't the default state be an empty string if the description is `null`?

With the change made, the defaultValue will always be empty.